api_test.go

```diff
@@ -440,9 +446,97 @@ func TestApiManyUpdateWithIpCheckHeaders(t *testing.T) {
 }
 
 func TestApiHealthCheck(t *testing.T) {
-	router := setupRouter(false, false)
+	router := setupRouter(false, false, "", "")
 	server := httptest.NewServer(router)
 	defer server.Close()
 	e := getExpect(t, server)
 	e.GET("/health").Expect().Status(http.StatusOK)
 }
+
+func TestApiCustomRegistration(t *testing.T) {
+	// Make sure a custom registration endpoint works
+	router := setupRouter(false, false, "", "custom-registration")
+	server := httptest.NewServer(router)
+	defer server.Close()
+	e := getExpect(t, server)
+	e.POST("/custom-registration").Expect().
+		Status(http.StatusCreated).
+		JSON().Object().
+		ContainsKey("fulldomain").
+		ContainsKey("subdomain").
+		ContainsKey("username").
+		ContainsKey("password").
+		NotContainsKey("error")
+
+	e = getExpect(t, server)
+	e.POST("/custom-registration").Expect().
```

Comment:
Was supposed to call normal registration and verify it is an error (probably 404?)